.github/workflows/test_micro.mamba.pm.yml

```diff
@@ -0,0 +1,57 @@
+name: Test micro.mamba.pm
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 10 * * *'
+  push:
+    paths:
+      - '.github/workflows/test_micro.mamba.pm.yml'
+  pull_request:
+    paths:
+      - '.github/workflows/test_micro.mamba.pm.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+defaults:
+  run:
+    shell: bash -el {0}
+
+jobs:
+  test_micro_mamba_pm:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {os: ubuntu-latest, shell: bash, shell-source-param: -i}
+          - {os: macos-latest, shell: zsh, shell-source-param: -i}
+          - {os: macos-latest, shell: bash, shell-source-param: -l}
+          - {os: windows-latest, shell: bash, shell-source-param: -l}
+    steps:
+      - name: Install Micromamba (${{ matrix.os }}, ${{ matrix.shell }})
+        shell: ${{ matrix.shell }}
```

Comment:
I don't think jinja stuff works in `shell:`. Did you test this action?

Reply:
<img width="1076" alt="image" src="https://github.com/mamba-org/micromamba-releases/assets/29506042/fb3eb38b-9d32-4896-a18b-a0a0e1aa9239">
looks like it's not working

Reply:
I did not since the CI wouldn't launch :sweat_smile: